lua/minimal_bookmarks/init.lua

```diff
@@ -162,6 +162,91 @@ local function add_bookmark(append, name)
     add_current_line_to_bookmarks(append, name)
 end
 
+
+local function is_current_line_bookmarked()
```

Comment:
What is the purpose of this function? I don't see it being used anywhere

Reply:
Aaah I was working on a thing to display something in the gutter, I'll remove it for now :)